tests/cql/CqlTypeOperatorsTest.xml

```diff
@@ -111,7 +111,7 @@
 	</group>
 	<group name="ToQuantity">
 		<test name="String5D5CMToQuantity">
-			<expression>ToQuantity('5.5 cm')</expression>
+			<expression>ToQuantity('5.5 \'cm\'')</expression>
```

Comment:
`5.5 cm` is not a valid string representation of a `Quantity`.  It needs the `'` delimiters around the unit.